README.md

```diff
@@ -81,58 +81,68 @@ These final equations are the ones used in the function **compute_bz**, which ca
 
 ## Installation 
 
-First, clone the repository
+- Clone the repository
 
 ```
 git clone https://github.com/shimming-toolbox/susceptibility-to-fieldmap-fft.git
+cd susceptibility-to-fieldmap-fft
 ```
 
-Navigate to the project directory
+- Create a virtual environnement 
 
 ```
-cd susceptibility-to-fieldmap-fft
+conda create --name <name of your environement> python=3.9
+conda activate <name of your environment>
+```
+
+- Go to the develop branch (temporary)
+
+```
+git checkout develop
```

Comment:
```suggestion
```

If you merge the branch, the branch to use will be `main`, not `develop`. Also, it will be the default branch, so no need to specify it, and this chunk should be removed.